stylesheet/style.css

```diff
@@ -0,0 +1,86 @@
+body{
+  background-color: #EAEAEA;
+}
+.header-container{
+  display: flex;
+flex-direction: column;
+flex-wrap: wrap;
+}
+ li{
+  display: inline;
+  font-size: 1.2rem;
+}
+.logo-bar {
+  background-color: #093C60;
+}
+.image-bar img{
+  height: 200px;
+  flex:2;
+}
+.logo-bar img{
+  height: 200px;
```

Comment:
I'd prefer using width instead of height to handle image sizing, sometimes height behaves strange